packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -552,14 +610,18 @@ export default class StatsAnalyzer extends EventsScope {
             LoggerProxy.logger.info(`StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} frames received`);
           }
 
-          if (this.statsResults.resolutions[mediaType].recv.framesDecoded === this.lastStatsResults.resolutions[mediaType].recv.framesDecoded || this.statsResults.resolutions[mediaType].send.framesDecoded === 0) {
+          if (this.statsResults[mediaType].recv.framesDecoded === this.lastStatsResults[mediaType].recv.framesDecoded || this.statsResults.resolutions[mediaType].send.framesDecoded === 0) {
             LoggerProxy.logger.info(`StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} frames decoded`);
           }
 
           if (this.statsResults.resolutions[mediaType].recv.framesDropped - this.lastStatsResults.resolutions[mediaType].recv.framesDropped > 10) {
             LoggerProxy.logger.info(`StatsAnalyzer:index#compareLastStatsResult --> ${mediaType} frames are getting dropped`);
           }
         }
+
+        // we are not calling emitStartStopEvents() for sending or receiving share because sharing is often started and stopped
```

Comment:
The old stats code has never sent these events for "share", so by not sending them now from this new code we are not changing any behavior.